doc/schemas/module_group_schema.json

```diff
@@ -52,6 +55,7 @@
           {"$ref": "#/definitions/pow"},
           {"$ref": "#/definitions/quadrant_selector"},
           {"$ref": "#/definitions/ridged_multi"},
+          {"$ref": "#/definitions/reciprocal"},
```

Comment:
`reciprocal` should come before `ridged_multi`.

Reply:
Didn't know how I missed that...